_includes/tutorials/sliding-windows/kstreams/code/Makefile

```diff
@@ -0,0 +1,13 @@
+STEPS_DIR := tutorial-steps
+DEV_OUTPUTS_DIR := $(STEPS_DIR)/dev/outputs
+TEMP_DIR := $(shell mktemp -d)
+SEQUENCE := "dev, test, prod, ccloud"
+
+tutorial:
+	rm -r $(DEV_OUTPUTS_DIR) || true
+	mkdir $(DEV_OUTPUTS_DIR)
+	cp $(STEPS_DIR)/dev/datagen-temperature-reading.avsc $(TEMP_DIR)
+	harness-runner ../../../../../_data/harnesses/sliding-windows/kstreams.yml $(TEMP_DIR) $(SEQUENCE)
+	bash -c "diff --strip-trailing-cr  <(tail -n 1 $(STEPS_DIR)/dev/expected-output.txt) <(tail -n 1 $(DEV_OUTPUTS_DIR)/actual-output.txt)"
+	bash -c "grep 'BUILD SUCCESSFUL' $(DEV_OUTPUTS_DIR)/unit-test-output.txt"
```

Comment:
@awalther28 added verification step here for tests cf https://github.com/confluentinc/kafka-tutorials/pull/690#discussion_r565622095

Reply:
Thanks @bbejeck !